metaflow/plugins/kfp/kfp.py

```diff
@@ -998,23 +740,67 @@ def build_kfp_dag(
                     METAFLOW_USER=METAFLOW_USER,
                 )
                 metaflow_run_id = f"kfp-{dsl.RUN_ID_PLACEHOLDER}"
-                step_op_args = dict(
-                    cmd_template=kfp_component.cmd_template,
-                    metaflow_run_id=metaflow_run_id,
-                    metaflow_configs=metaflow_configs,
-                    passed_in_split_indexes=passed_in_split_indexes,
-                    preceding_component_inputs=preceding_component_inputs,
-                    preceding_component_outputs=kfp_component.preceding_component_outputs,
-                    flow_parameters_json=flow_parameters_json
-                    if node.name == "start"
-                    else None,
+
+                command = [
+                    "bash",
+                    "-ec",
+                    kfp_component.init_cmd
+                    + " && python -m metaflow.plugins.kfp.kfp_step_function"
```

Comment:
This is no longer a `kfp_step_function` ...

Reply:
How does `kfp_orchestration_function` sound?

Reply:
To my understanding, we decided to stick with `kfp_step_function`. Let me know if `kfp_orchestration_function` or another name sounds clearer.